src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -20,18 +22,18 @@ namespace Microsoft.PowerFx.Functions
     // Due to .Net static ctor initialization, must place in a separate class from Library. 
     internal static class LibraryFlags
     {
-        public static readonly RegexOptions RegExFlags = RegexOptions.Compiled | RegexOptions.CultureInvariant;
+        public static readonly RegexOptions RegExFlags = RegexOptions.Compiled | RegexOptions.CultureInvariant | RegexOptions.IgnoreCase;
     }
 
     internal static partial class Library
     {
         private static readonly RegexOptions RegExFlags = LibraryFlags.RegExFlags;
 
-        private static readonly Regex _ampmReplaceRegex = new Regex("[aA][mM]\\/[pP][mM]", RegExFlags);
-        private static readonly Regex _apReplaceRegex = new Regex("[aA]\\/[pP]", RegExFlags);
-        private static readonly Regex _minutesBeforeSecondsRegex = new Regex("[mM][^dDyYhH]+[sS]", RegExFlags);
-        private static readonly Regex _minutesAfterHoursRegex = new Regex("[hH][^dDyYmM]+[mM]", RegExFlags);
-        private static readonly Regex _minutesRegex = new Regex("[mM]", RegExFlags);
+        private static readonly Regex _ampmReplaceRegex = new Regex("am\\/pm", RegExFlags);
```

Comment:
why the change in regexs?

Reply:
Carlos mentioned use ignorecase for easy reading so I updated existing regex as well. Is it ok? or you want me remove ignorecase tage from RegExFlags?

Reply:
Updated RegExFlags for ignorecase here